metadata.json

```diff
@@ -61,7 +61,7 @@
   "requirements": [
     {
       "name": "puppet",
-      "version_requirement": ">= 4.10.0 < 6.0.0"
+      "version_requirement": ">= 4.10.0 < 7.0.0"
```

Comment:
this should be bumped to 5.5.whatever now?

Reply:
For the sake of a separate changelog entry, I was going to do this in a separate PR later.